assets/css/dokan-admin-commission-suborder-metabox.css

```diff
@@ -0,0 +1,847 @@
+.dokan-order-metabox-items .inside {
```

Comment:
Overriding the all CSS is not good approach which may affect the future compatibility. It would be good if we can resolve it through the minimal changes.